test/e2e-common.sh

```diff
@@ -32,6 +32,15 @@ function add_sidecar_registry() {
     rm -f ${TMPF}.read
 }
 
+# Add an internal PyPI server as sidecar to the task so we can upload it directly
```

Comment:
I think you can move it to pre apply shell script https://github.com/tektoncd/catalog/pull/344/files#diff-8ae0ae624e44e02fcc2fc23eea1cf651  because i don't think many other tasks would need this functions (as opposed to the registry which need to be used by all the "builder" tasks)

Reply:
make sense, would move it. :+1: